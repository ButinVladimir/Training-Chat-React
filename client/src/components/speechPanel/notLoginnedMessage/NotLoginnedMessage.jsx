import React from 'react';
import './style.css';

function NotLoginnedMessage() {
  return (
    <div className="not-loginned-message">Login first to send messages</div>
  );
}

export default NotLoginnedMessage;
