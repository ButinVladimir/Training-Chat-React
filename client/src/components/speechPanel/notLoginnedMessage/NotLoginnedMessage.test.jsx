import React from 'react';
import { shallow } from 'enzyme';
import NotLoginnedMessage from './NotLoginnedMessage';

describe('NotLoginnedMessage', () => {
  it('renders without crashing', () => {
    const component = shallow(<NotLoginnedMessage />);
    expect(component).toMatchSnapshot();
  });
});
