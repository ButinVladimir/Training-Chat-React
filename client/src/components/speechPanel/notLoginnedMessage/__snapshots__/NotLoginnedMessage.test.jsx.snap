// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotLoginnedMessage renders without crashing 1`] = `
<div
  className="not-loginned-message"
>
  Login first to send messages
</div>
`;
